### DISCUSSION TITLE
trankplots() and precis() function

### DISCUSSION THREAD

**Discussion by Zhe-Jessica** [StatisticalRethinkingJulia/StatisticalRethinking.jl#156] [Q&A]
Hi, I notice that trankplots() and precis() function has been added to StatisticalRethinking.jl during the previous update. This documentation (see below) also mentioned this two function. 
https://stanjulia.github.io/Stan.jl/stable/WALKTHROUGH2/
I tested them in the 09-Chapter 9. Markov Chain Monte Carlo.ipynb, but these functions don't work. The error shows 'trankplots not defined' and 'no method matching precis'. I think these two functions are very useful. Is it possible to give an example code for how to use them?

**Comment by goedman** [comment]
Hi @Zhe-Jessica 

First of all, apologies for not answering this. Apparently I don't get a message when a new discussion is started (or I have missed it).

To help you, is it correct to assume that Stan (the binary of cmdstan) works on your system? E.g. you can run the examples in either Stan.jl or StanSample.jl? If not, we first need to get to that point and, in that case, on which platform do you work? Linux, MacOS or Windows?

Have you used any of the notebooks in earlier chapters of any of the StatisticalRethinking.jl based projects? Do you use the SR2StanPluto project or older versions?

Just let me know if I use terminology you are not familiar with.

Best,
Rob
